python-cache-demo/geo-service/geolocation_db_conn.py

```diff
@@ -0,0 +1,25 @@
+import mysql.connector
```

Comment:
Nice. btw the dependency install is already taken care of for each service in the corresponding requirements.txt file (https://github.com/MA3CIN/resilient-six/blob/main/python-cache-demo/infra-service/requirements.txt), i tested it for infra-service.